src/vue/navigation/Passport.vue

```diff
@@ -93,11 +96,32 @@ export default {
     },
   },
 
+  async created () {
+    setInterval(() => {
+      this.loadKycData()
+    }, this.timeUpdateAccountDetails)
+    this.loadKycData()
+  },
+
   methods: {
+    ...mapActions({
+      loadKyc: vuexTypes.LOAD_KYC,
+      loadAccount: vuexTypes.LOAD_ACCOUNT,
+    }),
+
     ...mapMutations({
       clearState: vuexTypes.CLEAR_STATE,
     }),
 
+    async loadKycData () {
```

Comment:
1. Do not use words like "data" or "info" in variable and function names
2. The function name is misleading cuz actually loads also account.